dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -67,6 +67,9 @@ export CHE_IMAGE_REPO=${CHE_IMAGE_REPO:-${DEFAULT_CHE_IMAGE_REPO}}
 DEFAULT_IMAGE_INIT="eclipse/che-init"
 export IMAGE_INIT=${IMAGE_INIT:-${DEFAULT_IMAGE_INIT}}:${CHE_IMAGE_TAG}
 
+DEFAULT_CHE_CLI_IMAGE="eclipse/che-cli"
+export CHE_CLI_IMAGE=${CHE_CLI_IMAGE:-${DEFAULT_CHE_CLI_IMAGE}}:${CHE_IMAGE_TAG}
```

Comment:
Should `CHE_CLI_IMAGE_TAG` be here?

Reply:
also, is default image is not "eclipse/che" (-cli being like an alias that we might drop)

Reply:
we use same tag for che init and cli by default it is nightlym if you want custom you have to build all 3 images

Reply:
@benoitf I would prefer to use che-cli because it is more obvious, usually guys think that eclipse/che is che-server not cli :)

Reply:
@sleshchenko ok, I;ve updated changes in such a way that allows to override INIT and CLI images with tags